ChakraCore Samples/Hello World/Linux_OSX/sample.cpp

```diff
@@ -4,8 +4,22 @@
 //-------------------------------------------------------------------------------------------------------
 
 #include "ChakraCore.h"
+#include <stdlib.h>
```

Comment:
The reason I ran into the issue is that I changed the standard compile flag. The examples currently use ```-std=c++0x```, and I was compiling against the flag ``-std=c++11``.